src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -644,6 +645,10 @@ public function mustRevalidate()
      */
     public function getDate()
     {
+        if (!$this->headers->has('Date')) {
+            $this->setDate(new \DateTime(null, new \DateTimeZone('UTC')));
```

Comment:
Thought that too, but since I was only moving this logic I didn't want to change it in the same time. Could it have any performance impact?